Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.xaml

```diff
@@ -1,6 +1,12 @@
-﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+﻿<ResourceDictionary 
```

Comment:
Sorry, I was accessing from my W10 mobile phone while out with family and couldn't see the diff very well. It looks like you aren't a fan of the indentation that I made to the Resource Dictionary xmlns declarations? Is that the concern? Or is the concern that I added two more xmlns declarations and a ignorable property ("d", "mc", and "mc:Ignorable")?